templates/sidebar.html

```diff
@@ -110,9 +110,10 @@
             </div>
             
         </a>
-        <nav class="collapse col-md-4 col-lg-3 d-md-block bg-light sidebar" id="dash-sidebar">
+        <nav class="collapse col-md-4 col-lg-3 d-md-block bg-white sidebar" id="dash-sidebar">
             <div class="sidebar-sticky pb-4 pb-md-0">
                 <ul class="sidebar-list nav flex-column text-center">
+                    <div class="card px-2 py-2 mt-4 shadow">
```

Comment:
This shadow looks too much, doesn't it? I think `shadow-sm` will be better here. Also, take care of the indentation here.